frontend/src/app/modules/boards/boards-root/boards-root.component.ts

```diff
@@ -3,7 +3,7 @@ import {BoardCacheService} from "core-app/modules/boards/board/board-cache.servi
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/efa080b533080e81ff35bb7f58518c16bf0b1632/frontend/src/app/modules/boards/boards-root/boards-root.component.ts#L3"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>